### ISSUE TITLE
Resolving response path from tutorial.

### ISSUE THREAD

**Issue by joshuaiz** [mitchgavan/serverless-shop-tutorial#1]
Great tutorial so thanks!

I'm getting an error trying to resolve the path in the response:

`/.netlify/functions/charge`

Where is this supposed to reside? In the tutorial you say to put the file to `/src/lambda/charge` but in the code block and here on GitHub it says `/.netlify/functions/charge` so I'm a bit confused as to where we are supposed to fetch the response from.

Can you clarify or am I missing something?

**Comment by mitchgavan** [comment]
Hi 👋 The code for the lambda function should reside in `src/lambda/charge`. When your function is deployed on Netlify, the response will be available from `/.netlify/functions/charge`. So a proxy (`src/setupProxy.js`)  is used to closely replicate what the final production environment will look like during development.

[This section](https://mitchgavan.com/react-serverless-shop/#setup-netlify-lambda) in the tutorial specifies how to setup the proxy correctly.

And you can find more info about why the proxy is required [here](https://github.com/netlify/netlify-lambda#using-with-create-react-app-gatsby-and-other-development-servers)

**Comment by joshuaiz** [comment]
Hey @mitchgavan yeah I found the docs so it makes sense now.

I'm just getting the following error locally while developing and can't figure out why:

[HPM] Error occurred while trying to proxy request /charge from localhost:3000 to https://localhost:9000 (EPROTO) (https://nodejs.org/api/errors.html#errors_common_system_errors)

I've looked through everything I can find about the proxy and setting it up locally. I have it set up exactly as they do here (with yarn): https://github.com/netlify/create-react-app-lambda

It tries to proxy it to `:9000` but something is blocking it. I don't have anything else on `9000`. I've searched for the error and nothing I've found so far has worked to fix it.

Any ideas?
